implementation/raport/generator/raport_generator.py

```diff
@@ -127,33 +80,11 @@ def generate_age_weight_chart(self):
     def generate_age_height_chart(self):
         return self.generate_chart(
             self.age_height_array,
-            self.child_age_height_array_X,
-            self.child_age_height_array_Y,
             'age [months]',
             'height [cm]',
             'growth_height'
         )
 
-    def generate_age_eye_width_chart(self):
-        return self.generate_chart(
-            self.age_eye_width_array,
-            self.child_age_eye_width_array_X,
-            self.child_age_eye_width_array_Y,
-            'age [months]',
-            'eye width [mm]',
-            'growth_eye_width'
-        )
-
-    def generate_age_upper_lip_height_chart(self):
-        return self.generate_chart(
-            self.age_upper_lip_height_array,
-            self.child_age_upper_lip_height_array_X,
-            self.child_age_upper_lip_height_array_Y,
-            'age [months]',
-            'upper lip height [mm]',
-            'growth_upper_lip_height'
-        )
-
     def generate(self, figures_to_save):
```

Comment:
Won't work on Unix (making me and Krzysiu sad) - "\\" are specific for Windows, use sth like
```python
output_dir = os.path.join(self.path, "saved_charts")
fig.savefig(os.path.join(output_dir, f'growth_{fig_type}_{next_number}.pdf'), dpi=100)
```